### DISCUSSION TITLE
Dreame L10s Ultra: Making it vacuum without washing

### DISCUSSION THREAD

**Discussion by okonet** [merdok/homebridge-miot#533] [Q&A] [answered]
I followed the instructions on how to enable room cleaning, but I could not find any reference on how to change the cleaning mode for the room. I can only change the amount of water / power but not to change the mode afaik. Does anyone know how to do this?

**Comment by merdok** [comment, marked as answer]
You could use an automation, where you first switch the mode and then start the desired room cleaning. Not sure if the mode can included in the room cleaning config directly.

**Reply by okonet** [reply]
That’s something I could try. Thanks!

**Reply by okonet** [reply]
Is it possible to send multiple commands with one switch? It would be great to have these switches rather than using the automations / shortcuts.

**Reply by merdok** [reply]
Currently not. Maybe something that can be added in the future...
